### DISCUSSION TITLE
Reopening merge requests, which were autoclosed due to repository connection problems

### DISCUSSION THREAD

**Discussion by Scaronthesky** [renovatebot/renovate#17423] [Request Help] [answered]
How do I reopen merge requests which were autoclosed by the Renovate bot? According to the logs the merge request for a specific dependency was closed by renovate because it had problems to access the repository. 

I took a look at the documentation but could not find out how to do this. I tried renaming the merge request or removing the labels but this does not work. The bot won't reopen the merge request or create a new one for the same dependency upgrade. 

We are using the Renovate bot on a self-hosted Gitlab installation.

**Comment by rarkins** [comment, marked as answer]
I would have hoped that if you manually reopen an autoclosed MR then Renovate would find it and reuse it. If that's not the case, can you capture the debug logs for the affected branch and paste here?

If the MR was autoclosed, it should had a `- autoclosed` suffix added to it so that it *wouldn't* block new MRs. If they are being blocked by an autoclosed MR then that's unexpected and also needs debugging.

**Reply by Scaronthesky** [reply]
Sorry, what we thought was a temporary connection issue turned out to be a permanent issue due to some recent changes in the network infrastructure. The bot had no chance to connect to the repository at all. :)  Thanks for the explanation.
